README.md

```diff
@@ -241,6 +241,30 @@ function handleSelection(value, event) {
 }
 ```
 
+**onChange** { Function }: A callback to run when the menu is opened or closed. It will be passed the `isOpen` state of the menu, e.g. `{ isOpen: true }`
```

Comment:
Happy to change to `onMenuToggle`.

Regarding `onMenuOpen` / `onMenuClose` that would have been overkill for our use case. It's easy enough for consumers either way. As an API, I like to have as few hooks as possible- easier to learn. This one hook can service both open / close, consumer can branch in their handler if needs be.
You have final say. Will respect if you prefer to split it up.